.github/workflows/genai-tools.yml

```diff
@@ -181,6 +181,7 @@ jobs:
           python -m pip install ${{ env.SRC_DIR }} -v ${{ needs.openvino_download.outputs.ov_wheel_source }}
           python -m pip install -r ${{ env.WWB_PATH }}/requirements.txt ${{ needs.openvino_download.outputs.ov_wheel_source }}
           python -m pip install git+https://github.com/huggingface/optimum-intel.git@main#egg=optimum-intel
+          pip install -U --force-reinstall datasets==3.1.0
```

Comment:
let's check whether it's still required https://github.com/openvinotoolkit/openvino.genai/pull/1484